e2e/LongHaul/device/CustomPayload.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Text.Json.Serialization;
+using System;
+
+namespace Microsoft.Azure.Devices.LongHaul.Device
+{
+    internal class CustomPayload
+    {
+        [JsonPropertyName("randomId")]
+        public Guid RandomId { get; set; }
+
+        [JsonPropertyName("currentTime")]
+        public DateTimeOffset CurrentTime { get; set; }
```

Comment:
```suggestion
        [JsonPropertyName("currentTimeUtc")]
        public DateTimeOffset CurrentTimeUtc { get; set; }
```

Reply:
I think an interesting thing we could do on the client is calculate the delay and include that in the response. We can also send the device's Utc.Now to the service, so it could calculate the delay as well. Log them as metrics.

Reply:
Yeah, I like this idea!